KLargestArrayInElement.java

```diff
@@ -0,0 +1,36 @@
+//Method 1- Using Min Heap
+class Solution {
+
+    //Time Complexity: 0(nlogk) where n is the length of the array and k is the kth element in array to be returned
+    //Space Complexity : o(nlogk) where n is the length of the array and k is the kth element in array to be returned
```

Comment:
Please check the space complexity.

Reply:
Again, it is correct as per me. Can you please point out the mistake?

Reply:
The size of the heap is always `k` here.

Reply:
So the space complexity will be just 0(n)?

Reply:
It will be `O(k)`

Reply:
Ohh Ok. Thank you for the clarification. Really appreciate it.